jwst/rscd/rscd_sub.py

```diff
@@ -80,16 +81,18 @@ def correction_skip_groups(input_model, group_skip):
         RSCD-corrected science data
     """
 
+    # Create output as a copy of the input science data model
+    output = input_model.copy()
```

Comment:
I don't understand why the copy, just to close.  Why not just use `input_model`?

Reply:
thank you for catching this!